feg/cloud/go/services/controller/test_utils/defaults.go

```diff
@@ -14,8 +14,9 @@
 package test_utils
 
 import (
-	"github.com/go-openapi/swag"
 	"magma/feg/cloud/go/services/feg/obsidian/models"
+
+	"github.com/go-openapi/swag"
```

Comment:
most of FeG source uses "std packages", "external packages", "magma packages" import order, a LF between LN17 & 18 would look more inline with the module convention

Reply:
Looking at a couple other go files in feg/cloud/go/ and things look like std>magma>external, so I'll leave it like this for now. But in any case the plan, in a future PR, is to standardize as std>external>magma (via goimports)